package.json

```diff
@@ -27,7 +27,7 @@
     "node-fetch": "^2.6.0",
     "protobufjs": "^6.8.8",
     "retry-request": "^4.0.0",
-    "semver": "^7.0.0",
+    "semver": "~7.0.0",
```

Comment:
could we just pin this at a known good version of semver? We intend to drop Node 8 support in the relatively near future.

Reply:
(but are waiting for it to GA on cloud functions).